backend/src/main/java/com/staccato/staccato/domain/StaccatoImages.java

```diff
@@ -14,7 +14,7 @@
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class StaccatoImages {
-    private static final int MAX_COUNT = 5;
+    private static final int MAX_COUNT = 8;
```

Comment:
```suggestion
    private static final int MAX_COUNT = 8;
    
```

`static` 변수라서 다른 곳의 컨벤션처럼 밑에 한줄 띄워야할 것 같아요!